### DISCUSSION TITLE
Intellij Codeceptjs plugin

### DISCUSSION THREAD

**Discussion by APshenkin** [codeceptjs/CodeceptJS#3517] [Show and tell]
Hi folks,

I implemented CodeceptJS plugin for Intellij products, so now you can run tests directly from IDEA for single test/single file/whole folder and see results in standard IDEA testing tab. 🚀
And most coolest thing – File Structure for tests now contains list of all tests in file and you can use fuzzy search there for quick navigation.

You can find it here: https://plugins.jetbrains.com/plugin/20581-codeceptjs-support
Github repo: https://github.com/APshenkin/intellij-codeceptjs

PRs are welcome ☺️

Happy testing,
Andrei

**Comment by amerryma** [comment]
Works great! Although I'm not sure why mocha-multi is defaulted to on.
